libbpf-rs/Cargo.toml

```diff
@@ -48,7 +48,7 @@ libbpf-rs = {path = ".", features = ["generate-test-files"]}
 log = "0.4.4"
 memmem = "0.1.1"
 plain = "0.2.3"
-probe = "0.3"
+probe = ">=0.3, <0.6"
```

Comment:
Let's just bump to `0.5` then? Nobody is really testing any version ranges, so this is hard to test for correctness. It's also very atypical for Rust projects from what I understand.